packages/grafana-ui/src/types/panel.ts

```diff
@@ -5,11 +5,16 @@ import { ScopedVars } from './datasource';
 
 export type InterpolateFunction = (value: string, scopedVars?: ScopedVars, format?: string | Function) => string;
 
-export interface PanelProps<T = any> {
+export type PanelOptions<T extends {}, S> = T & {
```

Comment:
Can we remove this State addition to the PaneOptions, not sure it's needed in this PR?

Reply:
It wasn't state but generic for series options :)